src/solvers/refinement/string_refinement.h

```diff
@@ -29,6 +29,7 @@ Author: Alberto Griggio, alberto.griggio@gmail.com
 #include <solvers/refinement/string_refinement_invariant.h>
 
 #define DEFAULT_MAX_NB_REFINEMENT std::numeric_limits<size_t>::max()
+#define DEFAULT_MAX_STRING_LENGTH (1 << 30)
```

Comment:
add a comment that explains that value (it's good to have it in the commit message, but even better to have it in the code)